packages/graphql-yoga/src/types.ts

```diff
@@ -50,31 +50,31 @@ export interface YogaInitialContext {
 
 export type CORSOptions =
   | {
-    origin?: string[] | string
-    methods?: string[]
-    allowedHeaders?: string[]
-    exposedHeaders?: string[]
-    credentials?: boolean
-    maxAge?: number
-  }
+      origin?: string[] | string
+      methods?: string[]
+      allowedHeaders?: string[]
+      exposedHeaders?: string[]
+      credentials?: boolean
+      maxAge?: number
+    }
```

Comment:
I think you did not run prettier, and somehow changed the formatting?

Reply:
Not sure 🤔 need to check